src/stories/select/Select.module.scss

```diff
@@ -100,26 +110,38 @@
 
     &[data-headlessui-state~="active"] {
         .radioCircle {
-            border-color: var(--pte-colors-contentPrimary);
+            border-color: var(--pte-new-colors-contentPrimary);
         }
     }
 
     &[data-headlessui-state~="checked"] {
         .radioCircle {
-            border: 5px solid var(--pte-colors-contentPrimary);
+            border: 5px solid var(--pte-new-colors-contentPrimary);
         }
     }
 
-    &[data-headlessui-state~="disabled"] {
+    &[data-disabled=true], &[data-headlessui-state~="disabled"] {
         pointer-events: none;
         cursor: default;
 
         &, & * {
-            color: var(--pte-colors-contentDisabled);
+            color: var(--pte-new-colors-contentDisabled);
         }
 
         .radioCircle {
-            border-color: var(--pte-colors-contentDisabled);
+            border-color: var(--pte-new-colors-contentDisabled);
+        }
+    }
+
+    &[data-status="error"] {
```

Comment:
this makes sense for now, but we should probably generally reconsider success/error states for radio buttons. not sure if just coloring the buttons is super intuitive

Reply:
Had talked to Lily about this, yeah she eventually wants to do some changes to our error/success states in general across components, but said that this was fine for now